backend/src/main/java/de/neuefische/backend/model/WaveDirection.java

```diff
@@ -0,0 +1,12 @@
+package de.neuefische.backend.model;
```

Comment:
Ich würde denken, dass du die ganzen Details nicht einzeln in Models haben müsstest, sondern vielleicht ein Model machst indem all die Details zusammen drin sind. Also ein Model bestehen aus WaveDirection, Waveheight etc.

Reply:
Ich habe mir das so abgeguckt :D und es macht möglicherweise auch Sinn, weil meine API Antwort sehr verschachtelt ist. So lässt sich das mit Hilfe des ObjectMappers automatisch mappen. 
Aber ich schau mal ob ich das evtl noch anpasse.